e7-damage-calc/artifacts.js

```diff
@@ -4,7 +4,8 @@ const artifactDmgType = {
   aftermath: 'aftermath',
   attack: 'attack',
   critDmgBoost: 'crit-dmg-boost',
-  flat: 'flat'
+  flat: 'flat',
+  burn: 'burn'
```

Comment:
let's make this one dot so we can re-use with bleed and bomb

```suggestion
  dot: 'dot'
```